.travis.yml

```diff
@@ -0,0 +1,43 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+language: java
+
+# blacklist
+branches:
+  except:
+    - asf-site
+    - gh-pages
+
+matrix:
```

Comment:
do we need all the extra junk? Surely the "branches: except" should be enough? why do we have gh-pages there?

Reply:
we don't need them. I was just copying the file from master. I will only keep branches:except.

I think the yml file was from `distributedlog`, which it has a `gh-pages`. we can remove it from here.

Reply:
removed unused content, only keep `branches`